test/solution_summary.jl

```diff
@@ -25,6 +25,7 @@ function test_empty_model()
 * Solver : No optimizer attached.
 
 * Status
+  Result             : 1 of 0
```

Comment:
1 of 0?

Reply:
By default, `result = 1`, and the `result_count` is `0`, so that's literally what your asking for with something like `primal_status(model)`. 

I guess we could make this `0 of 0`?

Reply:
Can we move `Primal status` and `Dual status` under `* Candidate solution`?
E.g.,
```
* Status
Result count: X
Termination status: ...

* Candidate solution (result #Y)
Primal status: ...
Dual status: ...
```

Reply:
Oh yeah, that's a better idea.

Reply:
Done